### PR TITLE
f/upgrade-podfile-settings

### PR DESCRIPTION
**Pull request by Jeremikas**
Apply changes to IOS project `Podfile` needed to resolve incompatibilities with latest build of CocoaPods & Expo.

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -429,43 +429,43 @@ SPEC CHECKSUMS:
...
-  EXTaskManager: c4b3dd6936b827b040899a33e9788b42b17d5c25
-  EXTaskManagerInterface: f9869b0f902ba7ccead16e0768c6dd75603163e0
+  EXPrint: 24a83979240b575a6809e2cc23ab760483d2a54b
+  EXReactNativeAdapter: 673eb78015f8814a37be8b9675d572cc8e9a5218
+  EXSegment: 890dd3c5b073ada171c723676d74a59c9c4e7dfb
+  EXSensors: 4076823ec927a22644c0c3fce920ead645c17e53
+  EXSensorsInterface: c35e76f4365a5168509c57dbdec737fa59b63a2f
+  EXSMS: e594b32323343c10ebc19f3a30831cd706bf58fd
+  EXTaskManager: 8c4307ca48f0096a35fb945a863668b87ffb4916
+  EXTaskManagerInterface: 5599889483ed14d28a715cceaedfaafb7409b2d0
```
Comment (Jeremikas) [inline ios/Podfile.lock]: Most (but not all) of the dependency checksums have changed. I'm assuming that this is possibly due to having run the `pod install` command with the latest CocoaPods and Expo binaries.

File: ios/si-scan---reports.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -343,7 +343,7 @@
 			files = (
 			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-si-scan---reports/Pods-si-scan---reports-resources.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-si-scan---reports/Pods-si-scan---reports-resources.sh",
```
Comment (Jeremikas) [inline ios/si-scan---reports.xcodeproj/project.pbxproj]: I'm also assuming that this change is due to the version change for CocoaPods.

File: ios/Podfile.lock (hunk 1 above)
Comment (Jeremikas) [reply ios/Podfile.lock]: This appears to indicate that newer|different versions of these dependencies were installed.

**Comment by ricog** [comment]
Markian replied and indicated "you need to have specifically 1.5.3 version of cocoa pods". Do you think your PR will resolve that restriction, or should be back down our cocoa pods install to 1.5.3?

**Comment by Jeremikas** [comment]
I'm not 100% certain at this point, which leads me to think we should probably downgrade to 1.5.3

**Comment by Jeremikas** [comment]
@ricog Based on the number of build warnings and errors I'm seeing, I'm opting to downgrade CocoaPods to 1.5.3.

**Comment by Jeremikas** [comment]
I'm thinking we can just close this. Merging it is probably a very bad idea.

**Comment by ricog** [comment]
Agreed.

**Pull request closed without merging**